app/(dashboard)/instances/components/InstanceForm.tsx

```
@@ -127,269 +127,291 @@ const InstanceForm = ({
-              }
-              break;
-            case "boolean":
-              if (data.requestParams[key] === "true") data.requestParams[key] = true;
-              else data.requestParams[key] = false;
-              break;
-          }
-        });
+                break;
+              case "float64":
+                if (data.requestParams[key] === "") break;
+                const output = Number(data.requestParams[key]);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)